host/core/pipeline/host_pipeline.hpp

```diff
@@ -20,7 +20,7 @@ class HostPipeline
     : public DataObserver<StreamInfo, StreamData>
 {
 protected:
-    const unsigned c_data_queue_size = 100;
+    const unsigned c_data_queue_size = 30;
```

Comment:
Was the change done to reduce the memory usage (or maybe perceived latency) for hosts with slower processing?

Reply:
Yes. @alex-luxonis